### PR TITLE
API: Code of Conduct Page

### PR DESCRIPTION
**Pull request by nverbic**
Closes https://github.com/Women-Coding-Community/wcc-api/issues/17

Description: 

Create API for Code of Conduct page. 

api_about.yml:
  - add  path: /code_of_conduct

schemas/attributes.yml:
  - add SimpleSection:
    - title
    - description/text
    - list of strings that represent bulleted paragraphs on the page - I am not sure about this solution - to be discussed.

 Example for code-of-conduct is removed from the  api_about.yml. 
 Examples are added to the components in attributes.yml.

TODO: Use Hero component without image

Screenshot (62)

Screenshot (63)

### CODE REVIEW

File: api_module_about.yml [hunk 1]
```diff
@@ -104,6 +104,51 @@ paths:
         default:
           $ref: 'schemas/responses.yml#/components/responses/Forbidden'
 
+  /code_of_conduct:
```
Comment (dricazenck) [inline api_module_about.yml:107]: I am not sure about `code_of_conduct` or `codeofconduct` @goelsonali @stepsen89 what do you think?

**Comment by dricazenck** [comment]
🚀 I think is really good to go, I am going to finish my changes and for me yours are good! 👏

File: api_module_about.yml (hunk 1 above)
Comment (goelsonali) [reply api_module_about.yml:107]: we should name as code-of-conduct
I read this about the best practices to name our end point.
https://blog.dreamfactory.com/best-practices-for-naming-rest-api-endpoints/
"It’s generally accepted that a hyphen is clearer and more user-friendly than using underscores (e.g. first_name) or camel case (e.g. firstName), which is discouraged due to its use of capital letters (see below)."

Comment (nverbic) [reply api_module_about.yml:107]: Great, I will update the name. ❤

**Review by goelsonali — APPROVED** [review]

**Review by dricazenck — APPROVED** [review]

**Comment by stepsen89** [comment]
We might need to think about a standard for BE/FE 

eg: 
description = always a paragraph 
items = ul 

In general looks fine. But this way you can't easily set something bold

**Pull request merged**